### DISCUSSION TITLE
Wish of Dropdown Component

### DISCUSSION THREAD

**Discussion by DSMejantel** [sqlpage/SQLPage#281] [Ideas]
Hello,
I've tried to select a dynamic link with a form and a select zone like this : 
```sql
SELECT 
    'form' as component,
    'etab_classes.sql?id=' || $id || '&classe_select=' ||$Classe AS action,
    'Valider' as validate,
    'green'           as validate_color; 

   SELECT 
'Classe' AS name, 
'select' as type, 
2 as width, 
json_group_array(json_object('label', classe, 'value', classe)) as options 
FROM (select distinct eleve.classe as classe, eleve.classe as value FROM eleve JOIN etab on eleve.etab_id=etab.id 
WHERE eleve.etab_id=$id  ORDER BY eleve.classe DESC);
```

It's ok and it does the job. But can we hope to  a dropdown more pretty to do that ?
For example with :
[https://tabler.io/docs/components/dropdowns](https://tabler.io/docs/components/dropdowns)

**Comment by lovasoa** [comment]
Support for autocomplete has been added and will be released in 0.20.3 :tada: 

You will be able to use a `select` input form with a `dropdown` property and create a beautiful input with autocomplete. 

https://sql.ophir.dev/documentation.sql?component=form#component
